add landing page for node-red learning resources docs (#2204)
FlowFuse/website: src/node-red/learn.md

---
eleventyNavigation:
  key: Learning Resources
  order: 1
meta:
   title: Node-RED Learning Resources
   description: Explore this comprehensive set of Node-RED learning resources, including how to set up Node-RED on various hardware platforms, its basic terminologies, integration of different databases, and more.
   keywords: node-red, node red, tutorial, learning resources, guide, node-red basics, node-red foundation. 
---

# Node-RED Learning Resources

> [Node-RED](/node-red/) is a low-code visual platform used for building IoT applications. This platform allows the integration of different hardware devices, API services, and technologies makes it exceptionally suited for industrial IoT applications.

Reviewer: ```suggestion
[Node-RED](/node-red/), the low-code visual programming platform, is becoming increasingly popular for optimizing and automating industrial processes. Enabling the integration of different hardware devices, API services, and technologies, it's exceptionally suited for a variety of industrial IoT applications.
```

> This documentation will help you get started with Node-RED, understand Node-RED Core Nodes, familiarize yourself with terminologies, and provide tutorials on integrating Node-RED with different databases, technologies, and protocols. Additionally, it covers how to run Node-RED on various hardware platforms.

Reviewer: ```suggestion
Here you will find all the resources you need to help you get started with Node-RED, understand Node-RED Core Nodes, familiarize yourself with terminologies, and tutorials on integrating Node-RED with different databases, technologies, and protocols. Additionally, we cover how to run Node-RED on various hardware platforms. 
```

> *Note: Currently, each section covers a few topics, but more content is expected to be added in the future.*

Reviewer: ```suggestion
*Note: We will be continuously adding new topics of content so be sure to bookmark and check back in.*
```

## Common Terminology

> This document contains some of the most common terminologies used in Node-RED communities and our resources. By learning these common terms, you can confidently navigate through most Node-RED-related resources and participate in basic Node-RED forum discussions.

Reviewer: ```suggestion
A good starting point is understanding the common terms used in Node-RED. You should be able to confidently navigate through most Node-RED-related resources and participate in Node-RED forum discussions.
```

[Node-RED Common Terminologies](/node-red/terminology/)

## Core Nodes

This section includes detailed documentation for each Node-RED core node, from configuration to use in real-world applications with examples.

> [Core Nodes Documentation](/node-red/core-nodes/)

Reviewer: ```suggestion
[Core Nodes](/node-red/core-nodes/)
```

## Hardware Devices

> This section of our documentation includes tutorials on setting up Node-RED on different hardware devices, tutorials covers configuration to practicle demostration along with troubleshooting tips.

Reviewer: ```suggestion
In this section, you will find tutorials on setting up Node-RED on different hardware devices. Each tutorial covers configuration, practical demos and troubleshooting tips.
```

[Setting Up Node-RED on Different Hardware](/node-red/hardware/)

## Peripheral Devices

This section includes step-by-step documentation on integrating Node-RED with various peripheral devices such as webcams, barcode scanners, printer, mic and more.

[Integrating Node-RED with Peripheral Devices](/node-red/peripheral/)

## Databases

> This section of our documentation includes tutorials on how to use different databases with Node-RED. Each document provides step-by-step instructions along with practical examples. This section covers various types of databases such as SQL, NoSQL, and time-series databases.

Reviewer: ```suggestion
This section includes tutorials on how to use different databases with Node-RED. Each tutorial provides step-by-step instructions along with practical examples. We cover a variety of databases such as SQL, NoSQL, and time-series databases.
```

[Node-RED Database Integration Guides](/node-red/database/)

## Communication Protocols

> This section of our documentation includes comprehensive tutorials on how to use different protocols with Node-RED for building applications, communicating with edge devices, and more. Each tutorial provides step-by-step instructions along with practical demonstrations.

Reviewer: ```suggestion
This section includes comprehensive tutorials on how to use different protocols with Node-RED for building applications, communicating with edge devices, and more. Each tutorial provides step-by-step instructions along with practical demos.
```

[Using Different Protocols for Building Applications with Node-RED](/node-red/protocol/)

## Integration Technologies

> This section of our documentation includes tutorials on how to utilize various technologies with Node-RED, such as implementing webhooks for task automation, creating REST APIs, GraphQL integrations, and more.

Reviewer: ```suggestion
In this section you will find tutorials on how to utilize various technologies with Node-RED, such as implementing webhooks for task automation, creating REST APIs, GraphQL integrations, and more.
```

[Using Different Technologies for Building Applications with Node-RED](/node-red/integration-technologies/)

## Notification Services

> This section of the documentation covers how to integrate various notification services with Node-RED to send real-time alerts and notifications. It includes step-by-step guides along with troubleshooting tips.

Reviewer: ```suggestion
In this section, we cover how to integrate various notification services with Node-RED to send real-time alerts and notifications. It includes step-by-step guides along with troubleshooting tips.
```
